public/app/features/serviceaccounts/ServiceAccountsListPage.tsx

```diff
@@ -194,6 +194,7 @@ export const ServiceAccountsListPageUnconnected = ({
             options={[
               { label: 'All', value: ServiceAccountStateFilter.All },
               { label: 'With expired tokens', value: ServiceAccountStateFilter.WithExpiredTokens },
+              { label: 'Managed', value: ServiceAccountStateFilter.External },
```

Comment:
We should add this behind a feature check I think :slightly_smiling_face:

Reply:
done